benches/benches/scripts.rs

```diff
@@ -15,20 +15,29 @@ fn bench_scripts(c: &mut Criterion) {
     let scripts: Vec<_> = walkdir::WalkDir::new(&scripts_dir)
         .into_iter()
         .filter_map(|e| e.ok())
-        .filter(|e| e.path().extension().is_some_and(|ext| ext == "js"))
+        .filter(|e| {
```

Comment:
nit: small preference, but can we document with a comment the ignore file behavior.